README.md

```diff
@@ -0,0 +1,63 @@
+<h1 align="center">
+  Mapeo Webmaps
+</h1>
+
+![](.github/screenshot.png)
+
+This is the repository for [Digital Democracy](digital-democracy.org/)'s [Mapeo Webmaps](https://maps.mapeo.app/) - a platform for Mapeo users to create and manage shares of maps via a public URL.
+
```

Comment:
We should mention Mapeo-Webmaps-Public, and how this differs from public. Perhaps being a little more specific about what each ones does